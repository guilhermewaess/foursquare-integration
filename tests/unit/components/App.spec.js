import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App', () => {
  let component;
  beforeEach(() => {
    component = shallowMount(App, {
      stubs: ['v-app', 'v-content', 'router-view'],
    });
  });
  it('should have name', () => {
    expect(component.name()).toEqual('App');
  });
  describe('snapshot', () => {
    it('should render', () => {
      expect(component.element).toMatchSnapshot();
    });
  });
});
