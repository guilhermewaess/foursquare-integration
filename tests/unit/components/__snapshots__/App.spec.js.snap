// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should render 1`] = `
<v-app-stub>
  <v-content-stub>
    <router-view-stub />
  </v-content-stub>
</v-app-stub>
`;
